src/styles/index.scss

```diff
@@ -1,3 +1,16 @@
+@import 'variables/variables';
+@import 'blocks/reset';
+
+html {
+  font-size: $font-size;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
 body {
-  margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
Note only:
Set standard logic for the tag only if all pages in your project will use similar logic.
It is better to hang the centering on a class, not a tag, because in the future there will be more and more pages.

In this case it works, but keep it in mind for the future